ci-operator/config/opendatahub-io/vllm/opendatahub-io-vllm-main.yaml

```diff
@@ -57,42 +47,14 @@ resources:
     requests:
       cpu: "16"
       memory: 24Gi
-  vllm-build-main-rocm:
-    limits:
-      cpu: "16"
-      memory: 35Gi
-    requests:
-      cpu: "16"
-      memory: 35Gi
 tests:
-- always_run: true
-  as: smoke-test
-  cluster: build05
-  commands: |
-    export VIRTUAL_ENV=/opt/vllm
-    export PATH=$VIRTUAL_ENV/bin/:$PATH
-
-    cd extras
-
-    bash smoke-test.sh
-  container:
-    clone: true
-    from: vllm-build-main-cuda
 - as: cuda-pr-image-mirror
   steps:
     dependencies:
       SOURCE_IMAGE_REF: vllm-build-main-cuda
     env:
       IMAGE_REPO: vllm
-      RELEASE_VERSION: cuda-pr
-    workflow: opendatahub-io-ci-image-mirror
-- as: rocm-pr-image-mirror
-  steps:
-    dependencies:
-      SOURCE_IMAGE_REF: vllm-build-main-rocm
-    env:
-      IMAGE_REPO: vllm
-      RELEASE_VERSION: rocm-pr
+      RELEASE_VERSION: cuda
```

Comment:
```suggestion
```

Reply:
we had already discussed about it in below slack thread :
https://redhat-internal.slack.com/archives/C07A8ECBTPB/p1725891663670459